### PR TITLE
Release 8.1.1

### PR DESCRIPTION
**Pull request by pravallikavadd**

### CODE REVIEW

File: jaspersoft-containers/Docker/scalableQueryEngine/README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ and all the deployment configurations are managed by Helm charts and Redis is us
 1. Run `cd <CONTAINER_PATH>`.
 1. Download a commercial edition of JasperReports Server WAR File installer zip into the current directory.
 1. Clone the jaspersoft-containers repository to the current directory.
-   `git clone git@github.com:TIBCOSoftware/js-docker.git`
+   `git clone git@github.com:tibco/jaspersoft-containers.git`
```
Comment (ysavoche) [inline jaspersoft-containers/Docker/scalableQueryEngine/README.md:49]: @pravallikavadd here repository link is bad, should be `git clone git@github.com:TIBCOSoftware/js-docker.git`

File: jaspersoft-containers/K8s/jrs/README.md [hunk 2]
```diff
@@ -207,10 +207,10 @@ These details are stored in Kubernetes secrets and used as environment variables
       AUDIT_DB_NAME:  audit-db-name
       AUDIT_DB_USER_NAME: audit-db-user-name
       AUDIT_DB_PASSWORD: audit-password
-      
-4. Build the docker images for TIBCO JasperReports® Server, and Scalable Query Engine (see the [Docker JasperReports Server readme](../../Docker/jrs#readme) and [Docker Scalable Query Engine readme](../../Docker/scalableQueryEngine#readme) ).
-5. Generate the keystore and copy it to the `k8s/jrs/helm/secrets/keystore` folder, see here for [Keystore Generation ](../../Docker/jrs#keystore-generation).
-6. Copy the TIBCO JasperReports® Server license to the `k8s/jrs/helm/secrets/license` folder.
+
+3. Build the docker images for TIBCO JasperReports® Server, and Scalable Query Engine (see the [Docker JasperReports Server readme](../../Docker/jrs#readme) and [Docker Scalable Query Engine readme](../../Docker/scalableAdhocWorker#readme) ).
```
Comment (ysavoche) [inline jaspersoft-containers/K8s/jrs/README.md:211]: link is wrong I think, should be as before - (../../Docker/scalableQueryEngine#readme)

File: jaspersoft-containers/K8s/jrs/README.md [hunk 3]
```diff
@@ -374,7 +374,7 @@ In case if you want to control traffic on AWS Application Load Balancer side ins
  
 ## Installation
 
-1. Clone the jaspersoft-containers ``git clone git@github.com:TIBCOSoftware/js-docker.git``.
+1. Clone the jaspersoft-containers ``git clone git@github.com:tibco/jaspersoft-containers.git``.
```
Comment (ysavoche) [inline jaspersoft-containers/K8s/jrs/README.md:377]: wrong repo

**Review by ysavoche — CHANGES_REQUESTED** [review]
@pravallikavadd please check issues

**Review by mbochenk — APPROVED** [review]

File: jaspersoft-containers/Docker/jrs/README.md [hunk 4]
```diff
@@ -231,12 +231,13 @@ If you plan to work with **default** JasperReports® Server and buildomatic setu
...
 
 - It uses haproxy as a load balancer and activemq as a cache replication. Before launching the application, make sure images are created successfully and repository DB setup is also completed.
+- Update the value of parameter 'replicas' in cluster-docker-compose.yml file depending upon how many containers you would want to create.
 
    `docker-compose -f cluster-docker-compose.yml up -d`
 
 - Wait for the application to start and access the application by using `host-name/jasperserver-pro` (port is not needed, haproxy is running on port 80).
 
- **Note:** Although an external repository DB is used, a PostgreSQL container is created, and it does not impact TIBCO JasperReports® Server. If you don't want to create a PostgreSQL container, then remove repository dependency from jasperserver-webapp-1 and jasperserver-webapp-2 services in cluster-docker-compose.yaml.
+ **Note:** Although an external repository DB is used, a PostgreSQL container is created, and it does not impact TIBCO JasperReports® Server. If you don't want to create a PostgreSQL container, then remove repository dependency from under jasperserver-webapp service in cluster-docker-compose.yaml.
```
Comment (ysavoche) [inline jaspersoft-containers/Docker/jrs/README.md:240]: @pravallikavadd please remove this whole NOTE section, cluster-docker-compose file now doesn't deploy repository

**Review by ysavoche — CHANGES_REQUESTED** [review]

File: jaspersoft-containers/Docker/jrs/README.md (hunk 4 above)
Comment (pravallikavadd) [reply jaspersoft-containers/Docker/jrs/README.md]: removed the "Note" section under "deploying the application in cluster mode."

Comment (ysavoche) [reply jaspersoft-containers/Docker/jrs/README.md]: yes, now it looks OK

**Review by ysavoche — APPROVED** [review]

**Comment by hguntupa-tibco** [comment]
Merging release-8.1.1 changes to main

**Pull request merged**